components/Sponsors.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <section id="sponsors">
+    <div class="section-title">
+      <h2>Sponsors</h2>
+      <hr />
+      <i>Bitcamp 2022 is proudly co-hosted by the CoStar Group!</i>
+    </div>
+    <div class="sponsor-container">
+      <template v-for="name in names" :key="name">
+        <SponsorCard :name="name" :tier="tiers[name]" :url="urls[name]" />
+      </template>
+    </div>
+    <em class="call-to-action"
+      >Interested in sponsoring us? Email us at
+      <a href="mailto:sponsorship@bit.camp">sponsorship@bit.camp</a> or
+      <a href="./sponsor.html">view our prospectus</a>.</em
+    >
+  </section>
+</template>
+
+<script lang="ts">
```

Comment:
Same here, I would use Vue 3 composition API. Then instead of defining `data()`, you can use refs. For example:
```
const names = ref<string[]>(['CMNS', 'UMD CS Department', 'UMIACS', 'Clark School of Engineering']);
```
You also won't need the `SponsorCard` import after switching.